Content.Server/Tools/Innate/InnateToolSystem.cs

```diff
@@ -1,90 +1,100 @@
+using System.Linq;
+using Content.Shared.Body.Part;
 using Content.Shared.Destructible;
 using Content.Shared.Hands.Components;
 using Content.Shared.Hands.EntitySystems;
 using Content.Shared.Interaction.Components;
 using Content.Shared.Storage;
 using Content.Shared.Tag;
+using Robust.Shared.Network;
 using Robust.Shared.Random;
 
-namespace Content.Server.Tools.Innate
+namespace Content.Server.Tools.Innate;
+
+/// <summary>
+///     Spawns a list unremovable tools in hands if possible. Used for drones,
+///     borgs, or maybe even stuff like changeling armblades!
+/// </summary>
+public sealed class InnateToolSystem : EntitySystem
 {
-    /// <summary>
-    ///     Spawns a list unremovable tools in hands if possible. Used for drones,
-    ///     borgs, or maybe even stuff like changeling armblades!
-    /// </summary>
-    public sealed class InnateToolSystem : EntitySystem
+    [Dependency] private readonly IRobustRandom _robustRandom = default!;
+    [Dependency] private readonly SharedHandsSystem _sharedHandsSystem = default!;
+    [Dependency] private readonly TagSystem _tagSystem = default!;
+
+    private List<string> _toSpawn = new();
```

Comment:
i feel like this should just be on the `InnateToolComponent` rather than just in the system.

Reply:
Done